booster_api/features/src/support/helpers.py

```diff
@@ -0,0 +1,256 @@
+import os, datetime
+import requests as req
+import json, jmespath, operator
+import constants
+
+count = 1
+
+def create_space_name(template="SDD"):
+    var = datetime.datetime.now()
+    var = var.isoformat().rsplit('.')[0]
+    space = constants.launch_detail.userid_primary + "-" + template + "-space-" + var
+    space = space.replace('@','-')
```

Comment:
I like this change. Thanks for bringing this in, Len. I ignored that someone's user ID could also be an email address.

Reply:
Thanks!